csp/gitprojectsettings.csp

```diff
@@ -94,6 +96,11 @@ body {
             set settings.mappedItemsReadOnly = 0
         }
 
+        set newRemote = $Get(%request.Data("remoteRepo",1))
+        if (newRemote '= ##class(SourceControl.Git.Utils).GetConfiguredRemote()) {
```

Comment:
This looks like it's going to replace an uncensored remote with a censored remote, breaking things when you change the settings on HCC.